sourcegraph/cody
# Chat UI polish

Various cleanups to the new chat UI implementation:

- Restyles the toolbar to better match VS Code
- Adds tooltips
- Fixes the sticky nav not staying stuck
- Switches back to vanilla shadcn/ui Collapsible by adding our own UI (composition over customisation of the vanilla shadcn/ui where possible)
- Animates collapsible transition
- Fixes button hover states and sizes in collapsible panel rows
- Fixes Account and Settings change changing/sticking when clicked

## Test plan

- Check storybooks
- Click around tabs in VS Code, trying different themes
- CI
